### PR TITLE
Bugfix/doorbell feedback

### PR DESCRIPTION
**Pull request by mhudson**
Doorbell widget wasn't displaying inside the app for logged in users. 
Renamed footer_small partial to avoid confusion.
Refactored Google Analytics embed to keep it DRY

### CODE REVIEW

File: app/views/layouts/_doorbell.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- Doorbell -->
+<script type="text/javascript">
+    window.doorbellOptions = {
+        "id": '6873',
+        "hideEmail": false,
+        "email": '<%= current_user.present? ? current_user.email : "" %>',
```
Comment (krishofer) [inline app/views/layouts/_doorbell.html.erb:6]: If you wanted to shorten these ternary expressions and keep the templates a little leaner, you could use safe navigation to guard against no `current_user`

```
<%= current_user.present? ? current_user.email : "" %>

<%= current_user&.email %>
```

File: app/views/layouts/_ga.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    (function (i, s, o, g, r, a, m) {
+        i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+            (i[r].q = i[r].q || []).push(arguments)
+        }, i[r].l = 1 * new Date(); a = s.createElement(o),
+            m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+    ga('create', 'UA-55037512-1', 'auto');
+    ga('send', 'pageview');
+</script>
+<!-- End Google Analytics -->
```
Comment (krishofer) [inline app/views/layouts/_ga.html.erb:12]: Nice extraction and DRY'ing up here!

Comment (krishofer) [reply app/views/layouts/_ga.html.erb:12]: Do we want to extract the analytics UA code into an env variable, so it doesn't fire in development environments?

Comment (krishofer) [reply app/views/layouts/_ga.html.erb:12]: Or simply only render the partial in production environments is another option

File: app/views/layouts/_doorbell.html.erb (hunk 1 above)
Comment (mhudson) [reply app/views/layouts/_doorbell.html.erb:6]: neat - I hadn't seen that ternary syntax before.

File: app/views/layouts/_ga.html.erb (hunk 2 above)
Comment (mhudson) [reply app/views/layouts/_ga.html.erb:12]: I think option #1 is best- make it an env var, so we can test analytics in staging (under a different UA code). We've also got the GA upgrade to do in next 9 months...

**Review by krishofer — APPROVED** [review]
Looks good - let's ship it 🚀

**Pull request merged**